src/modules.json

```diff
@@ -293,6 +294,9 @@
       "js_file": "js/net.js",
       "require": ["assert", "events", "stream", "tcp", "util"]
     },
+    "next_tick": {
```

Comment:
As I have pointed at https://github.com/Rokid/ShadowNode/pull/401/files#r229584228, this needs to be changed to `internal/process/next_tick`.